cmd/soroban-cli/src/commands/global.rs

```diff
@@ -1,11 +1,21 @@
-use clap::arg;
+use clap::{
+    arg,
+    builder::styling::{AnsiColor, Styles},
+};
 use std::path::PathBuf;
 
 use super::config;
 
+const USAGE_STYLES: Styles = Styles::styled()
+    .header(AnsiColor::Red.on_default().bold())
+    .usage(AnsiColor::Red.on_default().bold())
+    .literal(AnsiColor::Blue.on_default().bold())
+    .placeholder(AnsiColor::Green.on_default());
```

Comment:
I think we should probably go with the clap v3 default colours, which are more green yellow focused. The red blue is very red, and red is usually associated with errors, at least in a lot of CLIs and interfaces. I realise red means a variety of things across different cultures.

```suggestion
    .header(AnsiColor::Yellow.on_default())
    .usage(AnsiColor::Green.on_default())
    .literal(AnsiColor::Green.on_default())
    .placeholder(AnsiColor::Green.on_default());
```

Reply:
I agree. The green-yellow color scheme in clap v3 is a more familiar to developers and less alarming choice, especially given the common association of red with errors. It's a good balance between readability and has stellar color brand (yellow) as well.